en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
 
 ```
 8. TEXT="1-{(YEAR)}",\[YEAR],25%,60%,52%,9%
 10. TEXT="1-{(FACT)}",\[FACT],4.5%,68.5%,91%,30%,CENTER,WORDWRAP
 ```
 
-Line 8 is the script that writes the year on a range from the first to the last card with a 'year' field. Line 10 is the script that writes a description of the event on all of the cards, from the first to last card with a ‘fact’ field. So how does this script produce the cards we need to play _Timeline_?
+Line 8 of the script instructs nanDECK to write the year for every card that was given a year under the spreadsheet's 'year' field. Line 10 tells nanDECK to write a description of the event on every card that was given a fact under the spreadsheet's 'fact' field. How does this script produce the cards we need to play _Timeline_?
 
-Rather than use complicated scripting to ensure that the 'year' is only written on one side of a printed card, a simple solution is provide instructions to print everything explicited designated in the linked Excel spreadsheet. In the spreadsheet, the first 58 rows provide designated image, description, and year to be printed on the 'face' and the next 58 rows give instructions to generate the backs of the cards, using an image and only the ‘fact’ field, because the ‘year’ field is left empty.  
+Rather than use complicated scripting to ensure that the year only appears on one side of each printed card, we instead provide instructions to print everything that was explicited designated in the linked Excel spreadsheet. In the spreadsheet, the first 58 rows provide an image, a description, and a year, to be printed on the face of the card. The next 58 rows only provide an image and a description, because the 'year' field is left empty.  
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="Two pages of cards in a grid format. The left hand page of cards feature years printed in bold and the right hand side of cards are the same but without the year printed." caption="Figure 7: The simplest way to have _nanDECK_ generate a card side without the date being printed is to duplicate the information used to make cards without text in the date field." %}
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="Two pages of cards in a grid format. The left hand page of cards feature years printed in bold and the right hand side of cards are the same but without the year printed." caption="Figure 7. The simplest way for nanDECK to generate a card side without the date is to duplicate the information and remove the 'year' fied from one half." %}
```

Comment:
Here again, the alt-text does not seem to match the image.

Reply:
Figure 7 alt-text should read, "A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card. "